### DISCUSSION TITLE
Welcome to icu4x Discussions!

### DISCUSSION THREAD

**Discussion by sffc** [unicode-org/icu4x#413] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alolita** [comment]
Great to see discussions being enabled!

**Comment by srl295** [comment]
Steven R. Loomis. Icu/CLDR and node.js/ecma, trying to learn this rust thing.
